pkgs/tools/graphics/pdf-sign/default.nix

```diff
@@ -1,17 +1,19 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, substituteAll
 
 , python3
+
```

Comment:
This change is not necessary, but I won't complain

Reply:
I like having everything separated if they are used in a different context. But I can see why having just one element hanging out by itself would be weird.